### PR TITLE
ci: migrate snapshot publishing from CircleCI to GHA

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -187,6 +187,25 @@ jobs:
...
+        with:
+          cache-node-modules: true
+      - name: Setup Bazel
+        uses: angular/dev-infra/github-actions/bazel/setup@ba9b4487ced515e5b4d87edd681a3bd9792444d6
+      - name: Setup Bazel RBE
+        uses: angular/dev-infra/github-actions/bazel/configure-remote@ba9b4487ced515e5b4d87edd681a3bd9792444d6
+      - name: Install node modules
+        run: yarn install --frozen-lockfile
+      - run: echo "https://${{secrets.SNAPSHOT_BUILDS_GITHUB_TOKEN}}:@github.com" > ${HOME}/.git_credentials
+      - run: yarn build
```
Comment (devversion) [inline .github/workflows/ci.yml:206]: did we always build from scratch in CircleCI? or did we re-use the previously-built artifacts?

**Review by devversion — COMMENTED** [review]
I think we need to remove the CircleCI code as well. Also do we know something about the runner size if we perform a new build here?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (josephperrott) [reply .github/workflows/ci.yml:214]: Yes this is the same process as previously.

**Comment by josephperrott** [comment]
I did some testing and didn't find any real difference between different runner sizes.

**Review by devversion — APPROVED** [review]

**Comment by atscott** [comment]
This PR was merged into the repository by commit e12a10256cdc192d06116509969e8156edf59b20.

**Pull request closed without merging**